releasenotes/notes/fix-data-streams-timestamp-bug-16c6feb9ad8a7128.yaml

```diff
@@ -0,0 +1,3 @@
+fixes:
+  - |
+    data_streams: Fix Kafka & SQS integrations.
```

Comment:
A slightly more detailed description of the bugfix would help readers of the release notes understand whether they were affected by the issue.